packages/SwingSet/test/vat-admin/broken-vat.js

```diff
@@ -1,4 +1,4 @@
+/* global missing */
```

Comment:
`missing` is not a global. We're testing that reference to it generates an error.